test/math/QuaternionTest.ooc

```diff
@@ -54,12 +58,26 @@ QuaternionTest: class extends Fixture {
 		this add("multiplication", func {
 			expect(this quaternion0 * this quaternion1 == this quaternion3)
 		})
+		this add("dot product", func {
+			expect(this quaternion0 dotProduct(this quaternion1), is equal to(1410.25f) within(tolerance))
+		})
 		this add("scalarMultiplication", func {
 			expect((-1.0f) * this quaternion0 == -this quaternion0)
 		})
 		this add("norm", func {
 			expect(this quaternion0 norm, is equal to(65.5991592f))
 		})
+		this add("inverse", func {
+			inverse := this quaternion2 inverse
+			product := this quaternion2 * inverse
+			expect(product real == 1.0f)
+			expect(product x > -tolerance) //FIXME The standard "is equal to(0.0) within(tolerance)" does not work here?
```

Comment:
What about `expect(productReal as Float, is equal to(1.0f) within(tolerance))`? Not that I like the look of it, but I'm curious.